code/modules/mob/living/simple_animal/hostile/terror_spiders/prince.dm

```diff
@@ -0,0 +1,59 @@
+
+// --------------------------------------------------------------------------------
+// ----------------- TERROR SPIDERS: T3 PRINCE OF TERROR --------------------------
+// --------------------------------------------------------------------------------
+// -------------: ROLE: boss
+// -------------: AI: no special ai
+// -------------: SPECIAL: massive health
+// -------------: TO FIGHT IT: a squad of at least 4 people with laser rifles.
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/LightningBruiser
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/WakeupCallBoss
+// -------------: SPRITES FROM: FoS, http://nanotrasen.se/phpBB3/memberlist.php?mode=viewprofile&u=386
+
+/mob/living/simple_animal/hostile/poison/terror_spider/prince
+	name = "Prince of Terror spider"
+	desc = "An enormous, terrifying spider. It looks like it is judging everything it sees. Extremely dangerous."
+	spider_role_summary = "Boss-level terror spider. Lightning bruiser. Capable of taking on a squad by itself."
+	ai_target_method = TS_DAMAGE_BRUTE
+	egg_name = "prince spider eggs"
+
+	altnames = list("Prince of Terror","Terror Prince")
+	icon_state = "terror_queen"
+	icon_living = "terror_queen"
+	icon_dead = "terror_queen_dead"
+	maxHealth = 600 // 30 laser shots. 15 cannon shots
+	health = 600
+	melee_damage_lower = 15
+	melee_damage_upper = 25
+	move_to_delay = 5
+	ventcrawler = 1
+	loot = list(/obj/item/clothing/accessory/medal)
+
+	ai_ventcrawls = 0 // no override, never ventcrawls. Ever.
```

Comment:
this comment is useless, no shit it doesn't ventcrawl, you just set `ai_ventcrawls` to 0